### ISSUE TITLE
Request to append credits to the featured image

### ISSUE THREAD

**Issue by TerenceMilbourn** [Labs64/credit-tracker#22]
Append credits to the featured image as description or in a shortcode

**Comment by elohimito** [comment]
Is there any progress on this? Wondering if it's worth waiting for. Thanks.

**Comment by r-brown** [comment]
Hello @elohimito, as the request rate for this feature was very low we've deprioritized this. We'll try to enable this new feature in one of the next Credit Tracker releases.

**Comment by elohimito** [comment]
😭😭😭

**Comment by Jaace** [comment]
Is this not a duplicate of (or very similar to) https://github.com/Labs64/credit-tracker/issues/9 ?

**Comment by r-brown** [comment]
Hi @Jaace - yes, you are right, this issue is very similar to #9 
So it make sense to continue with #9 only.
I'll close this one.

**Comment by r-brown** [comment]
follow up is needed
```
Hi Developers,
after update to the recent version all of my blog thumbnails where gone.
Only after checking "Add credit to the post thumbnail (featured image)" did the thumnails reappear. Two problems with this:

1. What if I don't want credits beneath thumbnails?
2. The user agent stylesheet for "figure" tag, which was introduced with the update creates unwanted marging. The workaround here is of course to add custom css, but this still does not solve problem 1.

WP and all used plugins are up-to-date. I am using AVADA Theme on the website in question.

Best Regards, Guenon
```

Link: https://wordpress.org/support/topic/add-credit-to-the-post-thumbnail-featured-image/
